src/get_webhook_data.py

```
@@ -158,11 +169,56 @@ async def webhook(r: InputRequest):
             time.sleep(30)
         if counter > 3:
             break
+    
+    # Create and upload CSV file to GitHub
+    current_date = datetime.today().date().strftime("%Y-%m-%d")
```

Ruff (0.8.2)

`datetime.datetime.today()` used

(DTZ002)